.travis.yml

```diff
@@ -31,6 +31,10 @@ env:
     - GH_PROJECT_NAME: todomvc
     - CHROME_PATH: '/home/travis/build/tastejs/todomvc/chrome-linux/chrome'
 
+#update npm to latest stable version
+before_install:
+  - npm i -g npm
```

Comment:
this seems kinda weird :)

Reply:
I added this because the default npm version TravisCI is using is npm v5.1 while npm CI needs npm v5.8.
This is the only way I have in mind to change the default npm version to 5.8 as per https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Travis-CI-uses-npm
WDYT?

Reply:
hmm alright mind setting it to something exact then npm i -g npm@X.X.X ?

Reply:
@samccone done